frontend/packages/dev-console/src/components/ProjectsExistWrapper.tsx

```diff
@@ -34,7 +34,7 @@ const ProjectsExistWrapper: React.FC<ProjectsExistWrapperProps> = ({
     );
   }
 
-  return children();
+  return children;
```

Comment:
Why did you make this change? @vikram-raj

Reply:
I make this change. So, I do not need to shallow ```ProjectsExistWrapper``` children to check what is rendered.

Reply:
Discussed with andrew. Nothing wrong with this change. It's not like the render function was being passed any parameters.